packages/smart-wallet/src/marshal-contexts.js

```diff
@@ -9,7 +9,9 @@ const { Fail, quote: q } = assert;
  * For a value with a known id in the board, we can use
  * that board id as a slot to preserve identity when marshaling.
  *
- * @typedef {`board${Digits}`} BoardId
+ *  typedef {`board${Digits}`} BoardId // this was useful for development
```

Comment:
do you expect it to be again? the only reason to keep this line here is if you expect someone to get value out of re-enabling it.

I propose it be removed and the prefix be documented here, according to the resolution of the `prefix` option question

Reply:
Well, yes, I have some hope to return to the state where tsc was helping me partition the space of strings flying around in this module.

But I won't argue that it's worth the burden on everybody else reading this code to keep this comment around.